### ISSUE TITLE
Rule Change: Make no-confusing-arrow not conflict with Prettier

### ISSUE THREAD

**Issue by Garrett-R** [eslint/eslint#16662]
### What rule do you want to change?

no-confusing-arrow

### What change to do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
const xx = (aa) =>
  aa ? "pretty long string right here" : "another quite long string here";
```

### What does the rule currently do for this code?

It adds parentheses changing it to:

```js
const xx = (aa) =>
  (aa ? "pretty long string right here" : "another quite long string here");
```

However, Prettier will strip the parentheses, so this rule conflicts with Prettier.

### What will the rule do after it's changed?

It shouldn't complain about how Prettier formats the code or have an option to disable it for multi-line arrow functions.

### Participation

- [ ] I am willing to submit a pull request to implement this change.

**Comment by mdjermanovic** [comment]
Hi @Garrett-R, thanks for the issue!

Since Prettier requires parentheses around single-line conditionals ([Prettier demo](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAPbmC8mAFAIYCUBAfJpgEaYD8mYmSmAJgNwgA0IEABxgBLdMlAkATpIgB3AApSEaZCBIAbWSQCeKvrUkkwAazgwAyiQC2cADLCocZADMNaOPsMmz5gUYcA5sgwkgCuHiBwVrRw7OyxtiRQAaEkAXAAYhCSViQwIsmqJKEwELwgABYwVuoA6hXC8Gh+YHDmyo3CAG6N2qpgaHogDu6SMPKGAbkubhEAVmjY5oHqcACKoRDwM+rufH6So6q0JDHq5QKSDjC1wuwwFcgAHAAM+zLutYYCqpdwo10nHwAI6beATQQqFAkNAAWkcsVi5UkcFBwhREzS0yQrl2EXcVmEwTC+JWcAAgvkrrQSnB5HBJPZHDs9iA0GSNlsnDjZnwYKdbvdHkgAEx8wzCdSBADCECs2MiaAArOVQu4ACqnKG41ldcIASSg8Vg5jAVyE5KN5hg2lWLLgAF8HUA)), using the `no-confusing-arrow` rule with Prettier seems redundant, so you can simply disable this rule and let Prettier handle these parentheses.

**Comment by Susy1750** [comment]
I'm in support of what , #millos says will should allow traige to complete 💯 his kob

**Comment by Garrett-R** [comment]
Great point, thanks!
